src/sentry/static/sentry/app/components/sidebar/sidebarItem.tsx

```diff
@@ -130,7 +128,7 @@ const SidebarItem = ({
 
 export default ReactRouter.withRouter(SidebarItem);
 
-const getActiveStyle = ({active, theme}) => {
```

Comment:
We should try to avoid using imported `theme` when possible, using `theme` from props is preferred because it's through emotion's `ThemeProvider` - this is how we can have multiple themes (if we ever add multiple themes)